builtin/game/falling.lua

```diff
@@ -17,28 +19,25 @@ core.register_entity(":__builtin:falling_node", {
 		collisionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
 	},
 
-	node = {},
-	meta = {},
-
 	set_node = function(self, node, meta)
 		self.node = node
-		self.meta = meta or {}
+		self.meta = meta
 		self.object:set_properties({
 			is_visible = true,
-			textures = {node.name},
+			textures = {node.name}
```

Comment:
Removing commas is not necessary, it's actually better to leave them in to allow easier adding of new lines.

Reply:
????? a comma ????? seriously!